module/userExtension/helper.js

```diff
@@ -227,6 +227,20 @@ module.exports = class UserExtensionHelper {
         const allProgramIds = new Set();
         const allUserIds = new Set();
 
+        //delete invalid program operations from csv data 
+        for(let i= userRolesCSVData.length -1; i>=0; i--){
```

Comment:
@borkarsaish65 Please do not use I as a variable. please use some meaningful name